src/test/java/org/folio/search/service/converter/SearchDocumentConverterTest.java

```diff
@@ -54,12 +68,12 @@ void convertSingle_positive() {
     var resourceDescription = resourceDescription(getDescriptionFields(), List.of("$.language"));
 
     when(descriptionService.get(RESOURCE_NAME)).thenReturn(resourceDescription);
-    when(descriptionService.isSupportedLanguage("eng")).thenReturn(true);
+    when(languageConfigService.getAllSupportedLanguageCodes()).thenReturn(Set.of("eng"));
 
-    var actual = documentMapper.convert(eventBody);
+    var actual = documentMapper.convert(List.of(eventBody));
```

Comment:
name of method is convertSingle, why list of values passed?

Reply:
The test method is renamed. 

The `convert(ResourceEventBody)` has `private` access now because it's signature has been changed to `convert(IndexingConfig, ResourceEventBody)`

Reply:
So, I assume, indexing config should be passed to this method instead of changing method privacy and testName?

Reply:
This is now reverted.